flyway/src/main/resources/META-INF/native-image/io.micronaut.flyway/micronaut-flyway/reflect-config.json

```diff
@@ -0,0 +1,58 @@
+[
```

Comment:
I added the `reflect-config.json` generated with some output from `./gradlew -Pagent :test-suite-java:nativeTest`.